src/server/scripts/Northrend/Nexus/Nexus/boss_magus_telestra.cpp

```diff
@@ -18,6 +18,7 @@
 
 #include "ScriptMgr.h"
 #include "ScriptedCreature.h"
+#include "GameEventMgr.h"
```

Comment:
Not related to the PR, but instead compiled again with no PCH and added missing include so it Builds correctly.